src/FactSystem/FactMetaData.cc

```diff
@@ -1436,6 +1436,8 @@ FactMetaData* FactMetaData::createFromJsonObject(const QJsonObject& json, QMap<Q
 
     if (json.contains(_categoryJsonKey)) {
         metaData->setCategory(json[_categoryJsonKey].toString());
+    } else {
+        metaData->setCategory("Standard");
```

Comment:
@DonLakeFlyer I added this to preserve the current PX4 behavior. Are there cases (e.g. other components) where this is not desired?

Reply:
The "Standard" category concept is a PX4 firmware specific thing. The QGC default is actually "Other" which is setup up in the constructor. So this will change behavior I think moving ArduPilot params to a new category in some cases. Why is this needed?

Reply:
It's all fairly ancient history so I may have it somewhat wrong but it's something along those lines.

Reply:
Ok I make sure PX4 sets this then

Reply:
Make sense. I believe I had to add a workaround in the xml parser for the metadata for the missing categories ages ago.